cmd/lotus-seal-worker/main.go

```diff
@@ -450,7 +451,7 @@ func extractRoutableIP(timeout time.Duration) (string, error) {
 	env, ok := os.LookupEnv(minerMultiAddrKey)
 	if !ok {
 		// TODO remove after deprecation period
```

Comment:
^^ TODO?

Reply:
I'd kep it for a bit longer, it was present in the last testnet, so some people can have setups which use it